aws-rds-dbclusterparametergroup/src/main/java/software/amazon/rds/dbclusterparametergroup/BaseHandlerStd.java

```diff
@@ -119,6 +119,7 @@ public abstract class BaseHandlerStd extends BaseHandler<CallbackContext> {
             .build();
 
     protected HandlerConfig config;
+    protected RequestLogger requestLogger;
```

Comment:
This need to be set in every resource at handleRequest method. Here is an example: https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-rds/blob/master/aws-rds-dbinstance/src/main/java/software/amazon/rds/dbinstance/BaseHandlerStd.java#L398

Let stick with same variable name `requestLogger` across resources. feel free to rename already exist resources.